pkg/specgen/generate/container.go

```diff
@@ -444,9 +444,9 @@ func ConfigToSpec(rt *libpod.Runtime, specg *specgen.SpecGenerator, containerID
 		}
 	}
 
-	specg.HealthLogDestination = conf.HealthLogDestination
-	specg.HealthMaxLogCount = conf.HealthMaxLogCount
-	specg.HealthMaxLogSize = conf.HealthMaxLogSize
+	specg.HealthLogDestination = *conf.HealthLogDestination
+	specg.HealthMaxLogCount = *conf.HealthMaxLogCount
+	specg.HealthMaxLogSize = *conf.HealthMaxLogSize
```

Comment:
Doesn't this cause a nil deref when you do this on older containers that have nil set in their config.

Reply:
Good point